Makefile

```diff
@@ -32,7 +33,7 @@ dist/zbar.wast: dist/zbar.wasm
 	$(WASM2WAT) dist/zbar.wasm -o dist/zbar.wast
 
 dist/zbar.wasm: $(ZBAR_DEPS) src/module.c dist/symbol.test.o
-	$(EMCC) $(EMCC_FLAGS) -o dist/zbar.wasm src/module.c $(ZBAR_INC) \
+	$(EMCC) $(EMCC_FLAGS) -o dist/zbar.js src/module.c $(ZBAR_INC) \
```

Comment:
The reason to use `.wasm` not `.js` was because I want to minimize the js bundle. The original js bundle is too big and we only need a little lines of it.

I'm not sure how it the new `.js` bundle now, but I'd prefer to keep using `.wasm` if `src/instance.ts` is still useable.

Reply:
I did not manage to get the original configuration working with emscripten v3, so I used this approach. Although `wasm.js` is larger now, it saves some manual housekeeping work.
BTW, when compared with the increase in size of `zbar.wasm`, `wasm.js` does not appear that significant to me...

Reply:
When I started this project the js bundle is really big and didn't have any size optimization. Not sure how is going now. But we can do the cleaning if it is matter.

Then if we want to go this way, we may want to remove `load-browser.ts` because I believe that the js bundle has already handled it.